webpack/react_app/components/RegistrationExtension/RexPull.js

```diff
@@ -48,6 +49,12 @@ const RexPull = ({ isLoading, onChange, pluginValues, configParams }) => (
       {/* eslint-disable-next-line camelcase */
       options(configParams?.host_registration_remote_execution_pull)}
     </FormSelect>
+
+    <Alert type="info" isInline style={{ marginTop: '10px' }}>
+      {__(
+        'Please make sure that Smart Proxy is configured for pull-based remote execution.'
```

Comment:
@stejskalleos Can we use a variable instead of `Smart Proxy` so that it says `Capsule` in Satellite?

Reply:
`Smart Proxy` will automatically be translated to `Capsule` by foreman_theme_satellite.

Reply:
Oh, cool!